Themes/default/Errors.template.php

```diff
@@ -147,19 +145,19 @@ function template_error_log()
 		echo '
 							<a href="', $scripturl, '?action=admin;area=logs;sa=errorlog', $context['sort_direction'] == 'down' ? ';desc' : '', ';filter=error_type;value=', $error['error_type']['type'], '" title="', $txt['apply_filter'], ': ', $txt['filter_only_type'], '"><span class="generic_icons filter centericon"></span></a>
 							', $txt['error_type'], ': ', $error['error_type']['name'], '<br>
-							<a style="display: table-cell; padding: 4px 0; width: 20px; vertical-align: top;" href="', $scripturl, '?action=admin;area=logs;sa=errorlog', $context['sort_direction'] == 'down' ? ';desc' : '', ';filter=message;value=', $error['message']['href'], '" title="', $txt['apply_filter'], ': ', $txt['filter_only_message'], '"><span class="generic_icons filter"></span></a>
-							<span style="display: table-cell;">', $error['message']['html'], '</span>
+							<a class="error_message" href="', $scripturl, '?action=admin;area=logs;sa=errorlog', $context['sort_direction'] == 'down' ? ';desc' : '', ';filter=message;value=', $error['message']['href'], '" title="', $txt['apply_filter'], ': ', $txt['filter_only_message'], '"><span class="generic_icons filter"></span></a>
+							<span class="error_message">', $error['message']['html'], '</span>
 						</div>
 
-						<div style="float: left; width: 100%; padding: 4px 0; line-height: 1.6em; border-top: 1px solid #e3e3e3;">
-							<a style="display: table-cell; padding: 4px; width: 20px; vertical-align: top;" href="', $scripturl, '?action=admin;area=logs;sa=errorlog', $context['sort_direction'] == 'down' ? ';desc' : '', ';filter=url;value=', $error['url']['href'], '" title="', $txt['apply_filter'], ': ', $txt['filter_only_url'], '"><span class="generic_icons filter"></span></a>
-							<a style="display: table-cell;" href="', $error['url']['html'], '">', $error['url']['html'], '</a>
+						<div class="error_location">
```

Comment:
<img width="709" alt="screen shot 2017-12-06 at 15 25 14" src="https://user-images.githubusercontent.com/2927722/33690744-e162b10c-da99-11e7-97c3-1c5ac7f285e9.png">

**Current Error Log Table ^**

<img width="721" alt="screen shot 2017-12-06 at 15 25 19" src="https://user-images.githubusercontent.com/2927722/33690743-e145e194-da99-11e7-8feb-a8da7e6eefb4.png">

**Changed Error Log Table ^**

The additional cells help distinguish the different pieces of information. I find it harder to sort through the error log with everything aligned in one cell. Was this a deliberate style change?

Reply:
Hmm, did you include admin.css changes? That's not how it should look.

Reply:
Let me double check. Perhaps the .css was cached and I didn't force refresh after checking out the PR branch.

Reply:
And you are correct, thanks.